bundles/org.openhab.binding.hue/README.md

```diff
@@ -162,7 +217,62 @@ The group type also have an optional configuration value to specify the fade tim
 | fadetime  | Fade time in Milliseconds to a new state (min="0", step="100", default="400") |
 
 
-## Channels
+## Channels (CLIP v2)
+
+Normal devices support some of the following channels:
+
+| Channel Type ID       | Item Type          | Description                                                                           |
```

Comment:
Again, I think it would be quite useful to merge with table for v1 so users could clearly see the differences, i.e. which channels are only supported by v2.

Reply:
Hmm.. Maybe.. See my comment above _"if you are running API v2 please ignore the API v1 chapter below"_..